src/app/app-common/PageSource/PlainTextPageSource.cpp

```diff
@@ -64,6 +72,35 @@ PlainTextPageSource::PlainTextPageSource(
 PlainTextPageSource::~PlainTextPageSource() {
 }
 
+void PlainTextPageSource::ChangeFontSize(uint32_t newFontSize) {
+  if (newFontSize == mFontSize) {
+    return;
+  }
+
+  mFontSize = newFontSize;
+
+  auto dwf = mDXR.mDWriteFactory;
+  dwf->CreateTextFormat(
+    FixedWidthContentFont,
+    nullptr,
+    DWRITE_FONT_WEIGHT_NORMAL,
+    DWRITE_FONT_STYLE_NORMAL,
+    DWRITE_FONT_STRETCH_NORMAL,
+    newFontSize * RENDER_SCALE,
+    L"",
+    mTextFormat.put());
```

Comment:
IIRC this will raise an error when not the first call: you need to clear out mTextFormat before assigning to it. Just assigning `nullptr` will make this safe.